persistence-testkit/src/test/scala/org/apache/pekko/persistence/testkit/scaladsl/RuntimeJournalsSpec.scala

```diff
@@ -0,0 +1,123 @@
+package org.apache.pekko.persistence.testkit.scaladsl
```

Comment:
Can you add a standard ASF source header for this file?

https://github.com/apache/pekko/blob/main/project/AddMetaInfLicenseFiles.scala#L1-L16